taskt/UI/Forms/UIForm.cs

```diff
@@ -12,90 +12,88 @@
 
-            //(new Rectangle(0, BackgroundChangeIndex, this.Width, this.Height), Color.FromArgb(255, 214, 88), Color.Orange, System.Drawing.Drawing2D.LinearGradientMode.Vertical))
+            //(new Rectangle(0, BackgroundChangeIndex, Width, Height), Color.FromArgb(255, 214, 88), Color.Orange,
+            //LinearGradientMode.Vertical))
             //{
-            //    e.Graphics.FillRectangle(brush, 0, BackgroundChangeIndex, this.Width, this.Height);
+            //    e.Graphics.FillRectangle(brush, 0, BackgroundChangeIndex, Width, Height);
             //}
+
             var topColor = Color.FromArgb(49, 49, 49);
-            using (var brush = new System.Drawing.Drawing2D.LinearGradientBrush
-                  (new Rectangle(0, 0, this.Width, BackgroundChangeIndex), topColor, topColor, System.Drawing.Drawing2D.LinearGradientMode.Vertical))
+            using (var brush = new LinearGradientBrush
+                  (new Rectangle(0, 0, Width, BackgroundChangeIndex), topColor, topColor, LinearGradientMode.Vertical))
             {
-                e.Graphics.FillRectangle(brush, 0, 0, this.Width, BackgroundChangeIndex);
+                e.Graphics.FillRectangle(brush, 0, 0, Width, BackgroundChangeIndex);
             }
 
-            using (var brush = new System.Drawing.Drawing2D.LinearGradientBrush
+            using (var brush = new LinearGradientBrush
 
-            (new Rectangle(0, BackgroundChangeIndex, this.Width, this.Height), Color.SteelBlue, Color.LightSteelBlue, System.Drawing.Drawing2D.LinearGradientMode.Vertical))
+            (new Rectangle(0, BackgroundChangeIndex, Width, Height), Color.SteelBlue, Color.LightSteelBlue, LinearGradientMode.Vertical))
             {
-                e.Graphics.FillRectangle(brush, 0, BackgroundChangeIndex, this.Width, this.Height);
+                e.Graphics.FillRectangle(brush, 0, BackgroundChangeIndex, Width, Height);
             }
         }
 
-        private void InitializeComponent()
-        {
-            System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(UIForm));
-            this.SuspendLayout();
-            // 
-            // UIForm
-            // 
-            this.ClientSize = new System.Drawing.Size(284, 261);
-            this.Icon = ((System.Drawing.Icon)(resources.GetObject("$this.Icon")));
-            this.Name = "UIForm";
-            this.Load += new System.EventHandler(this.UIForm_Load);
-            this.ResumeLayout(false);
-
-        }
-
-        private void UIForm_Load(object sender, EventArgs e)
-        {
-        }
         public static void MoveFormToBottomRight(Form sender)
         {
-            sender.Location = new Point(Screen.FromPoint(sender.Location).WorkingArea.Right - sender.Width, Screen.FromPoint(sender.Location).WorkingArea.Bottom - sender.Height);
+            sender.Location = new Point(Screen.FromPoint(sender.Location).WorkingArea.Right -
```

Comment:
Same comment from earlier about making variables and passing the variables into the method instead of doing calculations in the method call.

Reply:
done